src/util/articleRenderers.ts

```diff
@@ -112,7 +112,7 @@ const renderMCSArticle = (json: any, dump: Dump, articleId: string, articleDetai
         const firstTocLevel = json.remaining.sections[0].toclevel;
         json.remaining.sections
             .forEach((oneSection: any, i: number) => {
-                if (oneSection.toclevel === firstTocLevel) {
+                if (oneSection.toclevel === firstTocLevel || oneSection.text.indexOf('class="NavFrame"')) {
```

Comment:
@bakshiutkarsha I'm not commenting on the general idea here, but only on your use of `indexOf`. The `indexOf` method returns -1 if the string you are searching does **not** contain the search string, not 0. Therefore, unless I've misunderstood what you're trying to do, I think you need:

`~oneSection.text.indexOf('class="NavFrame"')
`
**Note the tilde at the start.** This converts -1 to 0, and 0 to -1, etc. Therefore, if the text appears at position 0, the tilde converts 0 to -1 and the test will pass. But if `indexOf` returns -1 (not found), the tilde converts it to 0, and the test fails.

Reply:
PS It would probably be better to use the `classList.contains()` method for this test. It is supported on IE10+. See https://dev.caniuse.com/classlist .

Reply:
Also, your test would fail if an editor had written `class='NavFrame'` with single quotes or `class="dark NavFrame"`, hence why using `classList.contains()` is safer.